File: backend/src/main/java/org/example/backend/domain/tradecomment/repository/TradeCommentRepository.java
package org.example.backend.domain.tradecomment.repository;

import java.util.List;
import org.example.backend.domain.member.entity.Member;
import org.example.backend.domain.tradecomment.entity.TradeComment;
import org.springframework.data.jpa.repository.JpaRepository;

public interface TradeCommentRepository extends JpaRepository<TradeComment, Long> {

    List<TradeComment> findByTradeTradeId(Long tradeId);
    
    List<TradeComment> findByMember(Member member);

}

File: backend/src/main/java/org/example/backend/domain/tradecomment/repository/TradeCommentRepository.kt
package org.example.backend.domain.tradecomment.repository

import org.example.backend.domain.member.entity.Member
import org.example.backend.domain.tradecomment.entity.TradeComment
import org.springframework.data.jpa.repository.JpaRepository

interface TradeCommentRepository : JpaRepository<TradeComment, Long> {
    fun findByTradeTradeId(tradeId: Long): List<TradeComment>

    fun findByMember(member: Member): List<TradeComment>
}
